judgments/urls.py

```diff
@@ -39,18 +39,20 @@
         name="feed",
     ),
     re_path(
-        r"(?P<judgment_uri>.*/\d{4}/\d+)/data.pdf",
+        r"^(?P<judgment_uri>.*/\d{4}/\d+)/data.pdf$",
```

Comment:
Is it worth pulling the `(?P<judgment_uri>.*/\d{4}/\d+)` into a "this is what a judgment URI segment looks like" constant?

Reply:
This is probably another commit/PR.